### PR TITLE
Blazor prerendering language update

### LINKED ISSUE #14202: Unclear statement - Creating a new circuit containing all of the prerendered components
`Creating a new circuit containing all of the prerendered components that are part of the same page`

How can a circuit contain prerendered components? As I understood, a circuit is a websocket connection, it is a mode of transportation.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ce1440e7-febc-3fbc-09d7-7f47b740fd3b
* Version Independent ID: 31071595-7c71-b6f7-3856-74f78c5c7cdf
* Content: [Handle errors in ASP.NET Core Blazor apps](https://docs.microsoft.com/en-us/aspnet/core/blazor/handle-errors?view=aspnetcore-3.0#places-where-errors-may-occur)
* Content Source: [aspnetcore/blazor/handle-errors.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/handle-errors.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #14202

This is a small shift to the phrasing to move it away from "the circuit **_contains_** the components."

Thanks @Legends! 🎸

### CODE REVIEW

File: aspnetcore/blazor/handle-errors.md [hunk 1]
```diff
@@ -168,9 +168,9 @@ When a circuit ends because a user has disconnected and the framework is cleanin
 
 Blazor components can be prerendered using `Html.RenderComponentAsync` so that their rendered HTML markup is returned as part of the user's initial HTTP request. This works by:
 
-* Creating a new circuit containing all of the prerendered components that are part of the same page.
+* Creating a new circuit to update the prerendered components that are part of the same page.
 * Generating the initial HTML.
-* Treating the circuit as `disconnected` until the user's browser establishes a SignalR connection back to the same server to resume interactivity on the circuit.
+* Treating the circuit as `disconnected` until the user's browser establishes a SignalR connection back to the same server. When the connection is established, interactivity on the circuit is resumed and the components' HTML markup is updated.
```
Comment (danroth27) [inline aspnetcore/blazor/handle-errors.md]: This looks good.

File: aspnetcore/blazor/handle-errors.md [hunk 2]
```diff
@@ -168,9 +168,9 @@ When a circuit ends because a user has disconnected and the framework is cleanin
 
 Blazor components can be prerendered using `Html.RenderComponentAsync` so that their rendered HTML markup is returned as part of the user's initial HTTP request. This works by:
 
-* Creating a new circuit containing all of the prerendered components that are part of the same page.
+* Creating a new circuit to update the prerendered components that are part of the same page.
```
Comment (danroth27) [inline aspnetcore/blazor/handle-errors.md]: Hmm, we don't really update the components. We render them into the page as part of the response. Honestly, the relationship between the circuit and the prerendered components isn't super relevant.

Comment (danroth27) [reply aspnetcore/blazor/handle-errors.md]: How about this?

```suggestion
* Creating a new circuit for all of the prerendered components that are part of the same page.
```

**Comment by guardrex** [comment]
@danroth27 Updated. :eyes:

**Pull request merged**